live-examples/html-examples/image-and-multimedia/css/track.css

```
@@ -1,3 +1,6 @@
-video::cue {
+video {
+    width="250"
+}
```

Please add an empty line between style rule blocks
```suggestion
}

```